rust_src/src/windows.rs

```diff
@@ -672,6 +672,20 @@ pub fn window_live_p(object: Option<LispWindowRef>) -> bool {
     object.map_or(false, |m| m.is_live())
 }
 
+// Return new pixel size of window WINDOW.
+// WINDOW must be a valid window and defaults to the selected one.
+//
+// The new pixel size of WINDOW is the value set by the last call of
+// `set-window-new-pixel' for WINDOW.  If it is valid, it will be shortly
+// installed as WINDOW's pixel height (see `window-pixel-height') or pixel
+// width (see `window-pixel-width'). 
+
```

Comment:
Docstring comments should start with three slashes instead of two, and also the blank line between the comment and the function should be removed.

Reply:
Fixed.

Reply:
The blank line 681 still needs to be removed

Reply:
Ohhhhh, I thought you were talking about a blank commented line at 677, which was confusing to me because there were other blank commented lines in that file. My mistake!